docs/src/tutorials/tutorial_1.md

```diff
@@ -0,0 +1,127 @@
+# Tutorial 1: Run an Existing Model
+
+This tutorial walks through the steps to download, run, and view the output of an existing model.  There are several existing models public on Github, and for the purposes of this tutorial we will use [The Climate Framework for Uncertainty, Negotiation and Distribution (FUND)](http://www.fund-model.org), available on Github [here](https://github.com/fund-model/fund).
+
+Working through the following tutorial will require:
+
+- [Julia v1.0.0](https://julialang.org/downloads/) or higher
+- [Mimi v0.6.0](https://github.com/anthofflab/Mimi.jl) 
+- [Git](https://git-scm.com/downloads) and [Github](https://github.com)
+
+If you have not yet prepared these, go back to the main tutorial page and follow the instructions for their download.  
+
+### Step 1. Download FUND
+
+The first step in this process is downloading the FUND model.  Open a Julia REPL (here done with the alias command `Julia`) and navigate to the folder where you would like to download FUND.
+
+```
+Julia 
+cd("directory")
+```
+
+Next, clone the FUND repository from Github, enter the repository, and checkout the `next` branch which holds the Julia version of FUND.
+```
+git clone https://github.com/fund-model/fund.git
```

Comment:
This doesn't work in the julia REPL, right? Do we need some sort of indication what to execute in the REPL, and what somewhere else?